packages/core/content-releases/server/src/bootstrap.ts

```diff
@@ -107,22 +106,26 @@ export const bootstrap = async ({ strapi }: { strapi: LoadedStrapi }) => {
 
       async afterUpdate(event) {
         try {
-          // @ts-expect-error TODO: lifecycles types looks like are not 100% finished
           const { model, result } = event;
           // @ts-expect-error TODO: lifecycles types looks like are not 100% finished
           if (model.kind === 'collectionType' && model.options?.draftAndPublish) {
             const isEntryValid = await getEntryValidStatus(
               model.uid as Common.UID.ContentType,
-              result,
+              result as {
+                [key: string]: any;
+                id: StrapiEntity.ID;
+              },
               {
                 strapi,
               }
             );
 
+            const { id: targetId } = result as { id: StrapiEntity.ID };
```

Comment:
I'm not sure why you need this ?

Reply:
yep I think we can return this to how it was before